ospackage/man/saptune.8

```diff
@@ -38,14 +38,19 @@ saptune \- Comprehensive system optimisation management for SAP solutions
 all
 
 .SH DESCRIPTION
-saptune is a utility program that optimises your system according to recommendations/best practice guides written by SAP and SUSE, normally referred to as SAP Notes.
+saptune is designed to automate the configuration recommendations from SAP and SUSE to run an SAP application on SLES for SAP. These configuration recommendations normally referred to as SAP Notes. So some dedicated SAP Notes are the base for the work of saptune. Additional some best practice guides are added as Note definitions to optimise the system for some really special cases.
 
-saptune fully integrates with tuned(8), the tuned-profile name associated with this utility is "saptune".
+saptune can apply settings for a single Note definition or can set all settings for a predefined solution, which is a collection of several SAP Notes. Additional it can revert all these settings back to the previous system settings.
+
+It can verify, if the current system configuration is in sync with a specific Note definition or a defined solution and displays the differences.
+Additional it can simulate the setting of a single Note definition or a defined solution and will display the changes which will be done to the system.
```

Comment:
```sugestion
Additionally, it can simulate the setting of a single SAP Note definition or a defined solution and will display the changes which will be done to the system.
```

Reply:
Thanks, that's much more clearer

Reply:
see new commit https://github.com/SUSE/saptune/pull/7/commits/efe1570014676a635fad9d4ae3ea2646dfaea3f1